### ISSUE TITLE
HTTP JSON API integration with DAML Assistant

### ISSUE THREAD

**Issue by leo-da** [digital-asset/daml#2043]
@da-tanabe requested that HTTP JSON API should be integrated with DAML Assistant.
`daml start` should bring HTTP JSON API service up.

## Here is how we currently start it:
```
bazel run //ledger-service/http-json:http-json-binary -- \
    --ledger-host localhost --ledger-port 6865 --http-port 7575 \
    --max-inbound-message-size 4194304 --application-id HTTP-JSON-API-Gateway
```
## To build a fat jar:
```
$ bazel build //ledger-service/http-json:http-json-binary_deploy.jar
```

## ACs
1. `daml start` starts HTTP JSON API service, connecting to the default sandbox port
2. `daml json-api`  starts HTTP JSON API service

## Depends on:
- https://github.com/digital-asset/daml/issues/1631
- https://github.com/digital-asset/daml/issues/1630
- https://github.com/digital-asset/daml/pull/2751

**Comment by cocreature** [comment]
What exactly is involved in this? Do we just need to pass an additional flag to the sandbox?

**Comment by gerolf-da** [comment]
@cocreature: It's most likely a separate process that takes a ledger api endpoint (host, port) and offers its api on another port. The final user experience (i.e. integration into daml assistant, offer it as a library that can be integrated into sandbox, api server, ...) has not been decided yet.

**Comment by associahedron** [comment]
If it's a separate command you can do this yourself for now by passing `--on-start "command to start http server"` flag to `daml start`.

**Comment by cocreature** [comment]
Fixed by #2792
